src/mender-update/daemon/state_test.cpp

```diff
@@ -3109,7 +3112,16 @@ class StateDeathTest : public testing::TestWithParam<StateTransitionsTestCase> {
 				"artifact-name",
 				"--output-path",
 				ArtifactPath(),
-			});
+			};
+
+			auto scripts = MakeTestArtifactScripts(
+				GetParam(), tmpdir.Path(), path::Join(tmpdir.Path(), "state.log"));
+			for (auto &script : scripts) {
```

Comment:
nit: const

Reply:
Nah, not for local variables.

Reply:
I was thinking about the loop

Reply:
Yes, but it's also a local variable. It doesn't have any effect outside of this function, so I think it's overkill to put it everywhere.

Reply:
I've always thought of it as a readability thing, but I know you're a pragmatist here :p

Reply:
Gah, you make an annoyingly good argument! I admit I had not thought of it from that angle, and I guess it does express intent. Ok then, you convinced me, `const` it is!

Reply:
hahaha :tada: